src/app/lib/config/services/amharic.js

```diff
@@ -38,6 +38,10 @@ const service = {
     manifestPath: '/manifest.json',
     swPath: '/sw.js',
     frontPageTitle: 'ዜና',
+    theming: {
+      brandBackgroundColour: `${C_POSTBOX}`,
```

Comment:
Are we going to apply different styles for another components in the future? If so, should we extend the `theming` and add another key with the component like:
```
theming: {
      brand: {
            backgroundColour: `${C_POSTBOX}`,
            brandLogoColour: `${C_WHITE}`, 
      },
      ...
}

```

Reply:
I think there'll only be a couple of things - brand & navigation, so it shouldn't need extra nesting.